xilem/README.md

```diff
@@ -78,7 +78,7 @@ struct Counter {
     num: i32,
 }
 
-fn app_logic(data: &mut Counter) -> impl WidgetView<Counter> {
+fn app_logic(data: &mut Counter) -> impl WidgetView<Counter> + use<> {
```

Comment:
Obviously not a huge fan of this, but I've done some digging, and it seems unavoidable.

Now, there are macro based solutions that I'd be willing to consider, but that's by-the-by.